runblueprint/plans/test/test_planner.py

```diff
@@ -9,6 +9,12 @@ class TestPlannerFunctions(SimpleTestCase):
     def setUp(self):
         pass
 
+    def assert_listAlmostEqual(self, list1, list2):
```

Comment:
I believe this can be replaced with pytest.approx. https://docs.pytest.org/en/stable/reference.html#pytest-approx

Reply:
Oh, if we are not using Pytest yet, then ignore this comment. But let's start using Pytest whenever possible.